.github/actions/yarn-playwright-install/action.yml

```diff
@@ -11,9 +15,7 @@ runs:
           ~/Library/Caches/ms-playwright
           ~/.cache/ms-playwright
           ${{ github.workspace }}/node_modules/playwright
-        key: cache-playwright-${{ hashFiles('**/yarn.lock') }}
-        restore-keys: cache-playwright-
+        key: ${{ runner.os }}-playwright-${{ env.PLAYWRIGHT_VERSION }}
```

Comment:
Only bust playwright cache when the version actually changes. Not when any other dependency changes.